src/Controls/src/Core/Toolbar/Toolbar.Windows.cs

```diff
@@ -88,7 +88,7 @@ internal void UpdateMenu()
 
 		internal void OnToolbarItemPropertyChanged(object? sender, PropertyChangedEventArgs e)
 		{
-			if (Handler.PlatformView is not MauiToolbar wh)
+			if (Handler?.PlatformView is not MauiToolbar wh)
```

Comment:
LGTM, but, could you include a test to validate the scenario? Let me know if can help with something.

Reply:
> LGTM, but, could you include a test to validate the scenario? Let me know if can help with something.

Hi @jsuarezruiz  , Thank you for the review. The test case for this scenario is already present. We have re-enabled it for the Windows platform, which was previously disabled. Please let me know if there’s anything else to address.